charts/openmetadata/values.yaml

```diff
@@ -362,17 +363,17 @@ extraVolumeMounts: []
 
 extraInitContainers: []
 
-resources: {}
+resources:
 # We usually recommend not to specify default resources and to leave this as a conscious
 # choice for the user. This also increases chances charts run on environments with little
 # resources, such as Minikube. If you do want to specify resources, uncomment the following
 # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-# limits:
-#   cpu: 100m
-#   memory: 128Mi
-# requests:
-#   cpu: 100m
-#   memory: 128Mi
+  limits:
+    cpu: 1
+    memory: 2048Mi
+  requests:
+    cpu: 500m
+    memory: 256Mi
```

Comment:
We do not want to enable these limits and requests! Let's keep it in comments here!

Reply:
Acknowledged, updating the PR.